### PR TITLE
[zh-cn]: update the page of `<u>` element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

**Comment by T34-active** [comment]
@yin1999 PTAL

File: files/zh-cn/web/html/element/u/index.md [hunk 1]
```diff
@@ -112,7 +114,7 @@ u.spelling {
 <p>The class read <cite>Moby Dick</cite> in the first term.</p>
 ```
 
-{{EmbedLiveSample("使用 cite 元素", 650, 80)}}
+{{EmbedLiveSample("使用_cite_元素", 650, 80)}}
```
Comment (yin1999) [inline files/zh-cn/web/html/element/u/index.md:117]: please revert this, actually, I prefer the previous one (although both of them are correct). See: https://github.com/orgs/mdn/discussions/448#discussioncomment-6974683

File: files/zh-cn/web/html/element/u/index.md [hunk 2]
```diff
@@ -129,41 +131,65 @@ cite {
 }
 ```
 
-{{EmbedLiveSample("为 cite 元素添加样式", 650, 80)}}
+{{EmbedLiveSample("为_cite_元素添加样式", 650, 80)}}
```
Comment (yin1999) [inline files/zh-cn/web/html/element/u/index.md:134]: idto

File: files/zh-cn/web/html/element/u/index.md (hunk 1 above)
Comment (T34-active) [reply files/zh-cn/web/html/element/u/index.md:117]: got it

**Review by yin1999 — APPROVED** [review]

**Comment by T34-active** [comment]
@yin1999 Thank you!

**Pull request merged**